lib/src/ansi_code.dart

```diff
@@ -0,0 +1,278 @@
+// Copyright 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io' as io;
+
+/// The type of code represented by [AnsiCode].
+class AnsiCodeType {
+  /// The name of the type.
```

Comment:
Explain what the property is used for. Who uses it, when and why?

Explaining what it *is* should be covered by the type and the name, so you only need to say that if the name needs elaboration (which is reasonable for complex things that would be too long if the name explained everything).

If you don't know how the name is used, maybe it's not needed.
(That is, I can't see why the name is needed, but then, I don't see a good reason for the type at all).